bucket/lemon.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": "0.2.2",
+  "description": "A tiny judging environment for OI contest based on Lemon + LemonPlus.",
+  "homepage": "https://github.com/iotang/Project_LemonLime/",
+  "license": {
+    "identifier": "GPL-3.0"
+  },
+  "installer": {
+    "script": [
+      "Get-ChildItem -Path $dir\\5.15.0-desktop-win64_msvc2019_64\\* -Recurse | Move-Item -Destination $dir\\",
+      "Remove-Item $dir\\5.15.0-desktop-win64_msvc2019_64"
+    ]
+  },
```

Comment:
```
"extract_dir": "5.15.0-desktop-win64_msvc2019_64",
```
We can use this instead.
https://github.com/lukesampson/scoop/wiki/App-Manifests#user-content-optional-properties:~:text=extract_dir%3A%20If%20url%20points%20to%20a,just%20the%20directory%20specified%20from%20it.